eemeter/caltrack/hourly.py

```diff
@@ -147,10 +147,20 @@ def caltrack_hourly_prediction_feature_processor(
 
 def fit_caltrack_hourly_model_segment(segment_name, segment_data):
     def _get_hourly_model_formula(data):
-        bin_occupancy_interactions = "".join(
-            [" + {}:C(occupancy)".format(c) for c in data.columns if "bin" in c]
-        )
-        return "meter_value ~ C(hour_of_week) - 1{}".format(bin_occupancy_interactions)
+        if np.sum(data.loc[data.weight > 0].occupancy) == 0:
```

Comment:
Should this also account for the case with occupancy all `1`?

Reply:
Good point. It shouldn't happen with the default CalTRACK value for `threshold` because the majority of hours will have False occupancy. But we should address this in there, in case someone uses a different (lower than 0.5) threshold. I just pushed another commit with the extra condition